### ISSUE TITLE
Logged In User is defaulting to System Cron Jobs for all users that log into the Admin Client

### ISSUE THREAD

**Issue by TheIPKeeper** [boxbilling/boxbilling#1027]
### What happened?

After each staff member has logged into the Admin section of the system. The top left of the screen is showing as "Hi, System Cron Job". I setup a new staff member as a test and then logged in with the new member that has no permissions to anything. 
However, the logged in user still has permissions to everything on the system.

If I go to Activity --> Staff Login History. I can see that the users that have logged in are the test staff member that I have setup. When I create new Announcements, then this is reflecting as posted by System Cron Job

### BoxBilling Version

v4.21

### What version of PHP are you running?

PHP 7.4

### Additional Notes

I am current using BoxBilling on Cpanel v98.06, Centos v7.9

**Comment by BenNottelling** [comment]
Interesting, it sounds like on the backend BoxBilling may be incorrectly pulling account info for staff from the cron job account info.
And this was happening on both version `4.21` as well as `4.22.beta-1`?

**Comment by TheIPKeeper** [comment]
Correct. I upgraded to 4.22.beta-1 to see if this resolved the issue. but it does not.

**Comment by BenNottelling** [comment]
@TheIPKeeper had you run cron since creating the account?
I had seen the same problems as before, but noticed today that it was correctly showing my name:
![image](https://user-images.githubusercontent.com/17304943/144723397-56c45e78-d4d6-4e77-a2d5-6a286a4c837f.png)

I usually don't end up running the cron tab since I only run a BB site for development, but in this case I have
